pymc4/distributions/continuous.py

```diff
@@ -110,6 +111,67 @@ def _init_distribution(conditions):
         return tfd.Normal(loc=loc, scale=scale)
 
 
+class GeneralizedNormal(ContinuousDistribution):
+    r"""Univariate generalized normal random variable.
+
+    The pdf of this distribution is
+
+    .. math::
+
+       f(x \mid \mu, \alpha, \beta) = 
+           \frac{\beta}{2 \Gamma(1/\beta)}
+           \exp(-(|x - \mu| /\alpha)^\beta)
+
+    .. plot::
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        import scipy.stats as st
+        plt.style.use('seaborn-darkgrid')
```

Comment:
Hi @kori73 , thanks for your contribution! PyMC4 does not longer requieres seaborn. This should be plt.style.use('arviz-darkgrid'). If you are planning to keep adding distributions you may change this in your next PR.

Reply:
Sure, I will do that in my next PR.